framework/include/cppmicroservices/LDAPFilter.h

```diff
@@ -130,6 +130,10 @@ class US_Framework_EXPORT LDAPFilter {
    *        in the match.
    * @return <code>true</code> if the bundle's properties match this
    *         <code>LDAPFilter</code> <code>false</code> otherwise.
+   * @throws std::runtime_error If the number of keys of the <code>bundle</code>
+   *         exceeds the value returned by std::numeric_limits<int>::max().
+   * @throws std::runtime_error If the <code>bundle</code> contains case variants
```

Comment:
This can never happen, since the Match() function operates on the bundles headers, which is a case insensitive map already.